packages/gatsby-theme-docs/src/components/release-notes-subscribe-links.js

```diff
@@ -34,12 +27,6 @@ const linkStyles = css`
 
 const ReleaseNotesSubscribeLinks = () => (
   <SpacingsStack scale="xs">
-    <ExternalSiteLink href={withPrefix('/releases/feed.xml')} css={linkStyles}>
```

Comment:
I think we should remove the RSS link too, it's the one pointing to the original feed.xml file.